deploy/deployment.yaml

```diff
@@ -91,6 +91,8 @@ objects:
   - kind: Route
     apiVersion: v1
     metadata:
+      annotations:
+        haproxy.router.openshift.io/hsts_header: max-age=31536000
```

Comment:
Hm... I don't see any difference with this annotation or without it. What am I doing wrong? :)
Even if I also add `insecureEdgeTerminationPolicy` I still don't see auto redirects if I type `http:` explicitly in my Chrome.
Testing it in minishift.

Reply:
`insecureEdgeTerminationPolicy: Redirect` actually does the trick. And we already have it used in the route in toolchain-infra script for crt cluster: https://github.com/codeready-toolchain/toolchain-infra/blob/master/config/reg_service_route.yaml#L16

I think we don't have to use this redirect for the "internal" route we specify in deploy/deployment.yaml.

So, I'm closing it but please reopen if you think we still need to change something here.

Reply:
ok, thanks @alexeykazakov